src/components/screens/ExperimentView.vue

```diff
@@ -84,11 +84,32 @@
           </div>
         </div>
         <div v-else>No associated secondary publications.</div>
-        <div v-if="item.keywords && item.keywords.length > 0">
+        <div v-if="item.keywords && Object.keys(item.keywords).length > 0">
           <div class="mave-score-set-section-title">Keywords</div>
           <div class="mave-score-set-keywords">
-            <Chip :label="keyword" />
-            <a v-for="(keyword, i) of item.keywords" :key="i" :href="`${config.appBaseUrl}/search?search=${keyword}`"><Chip :label="keyword"/></a>
+            <li v-for="(keyword, index) in item.keywords" :key="index">
+              {{ keyword.keyword.key }}
+              <!--Present local database keyword description-->
+              <i class="pi pi-info-circle" style="color: green; cursor: pointer;" @click="showDialog(index)"/>
+              <Dialog v-model:visible="dialogVisible[index]" modal :header="keyword.keyword.key" :style="{ width: '50vw' }" :breakpoints="{ '1199px': '75vw', '575px': '90vw' }">
+                <p class="m-0">
+                  {{ keyword.keyword.description }}
+                </p>
+              </Dialog> : <a :href="`https://www.mavedb.org/search/?keywords=${keyword.keyword.value}`">{{ keyword.keyword.value }}</a>
```

Comment:
It doesn't seem like there is a mechanism in the score set search to search by keywords on associated experiments. We either want to add that or remove this link, since right now it will just link to a generic search when clicked.

Reply:
In search_score_set function, I added this part. My understanding for it is the associated score sets will be returned if users type keywords. However, it's online one so I can't test it and I thought just put it first and see how it's going. 
<img width="596" alt="image" src="https://github.com/user-attachments/assets/ddef7129-ae3a-4b79-a887-1094e4280165">

Reply:
I think that will allow us to load the score sets matching our filter logic, but we would still need to add the filter for keywords and add the . 

So in `src/mavedb/view_models/search.py` we would need to add a keywords list to the search model:
```
class ScoreSetsSearch(BaseModel):
    published: Optional[bool]
    targets: Optional[list[str]]
    target_organism_names: Optional[list[str]]
    target_types: Optional[list[str]]
    target_accessions: Optional[list[str]]
    authors: Optional[list[str]]
    databases: Optional[list[str]]
    journals: Optional[list[str]]
    publication_identifiers: Optional[list[str]]
    keywords: Optional[list[str]]
    text: Optional[str]
```

And then in `src/mavedb/lib/score_sets.py` we'd have to add a block that queries the experiment keywords:
```
if search.keywords:
    query = query.filter(
        Experiment.keyword_objs.any(
            ControlledKeyword.value.in_(search.keywords)
        )
    )
```

I haven't tested that, but would be happy to look together if you wanted to add it in.